### DISCUSSION TITLE
Add vertical-align sub and super values

### DISCUSSION THREAD

**Discussion by TCatinaud** [tailwindlabs/tailwindcss#2909] [Ideas]
I sometimes need the values ​​`sub` and` super` (especially `sub`) in the case of an inline-block. Can we add them?

**Comment by MathGeniusJodie** [comment]
it's an easy pull request
https://github.com/tailwindlabs/tailwindcss/blob/master/src/plugins/verticalAlign.js

**Reply by TCatinaud** [reply]
Done (PR #2910 ), thanks!
